aws/autoscaler/main.tf

```diff
@@ -4,7 +4,7 @@ data "aws_eks_cluster" "cluster" {
 
 data "aws_eks_node_group" "node_group" {
   cluster_name    = var.cluster_name
-  node_group_name = "${var.cluster_name}-deployment-node-group"
+  node_group_name = "${var.cluster_name}-deployment"
```

Comment:
the name was incorrect

Reply:
Because you changed it in both places :-)

Reply:
yes, got it only after... but now it's okay